src/java/main/org/apache/zookeeper/server/quorum/Observer.java

```diff
@@ -63,10 +63,10 @@ void observeLeader() throws Exception {
         zk.registerJMX(new ObserverBean(this, zk), self.jmxLocalPeerBean);
 
         try {
-            InetSocketAddress addr = findLeader();
```

Comment:
I am not sure what this change of addr from type InetSocketAddress to QuorumServer is for. I may miss something though.